default.nix

```diff
@@ -35,7 +35,12 @@ in rec {
     sha256 = "1wnz52m0ilsi18yrrv4iw2629ja0kjhqf53aldyrfb32ri8zcjxg";
   };
 
-  v1_6_2 = callPackage ./clever-tools {};
+  v1_6_2 = mkOlderVersion {
+    rev = "e3b8f435cbb59fd64743ab495dcffb3496fee8e5";
+    sha256 = "1wnz52m0ilsi18yrrv4iw2629ja0kjhqf53aldyrfb32ri8zcjxg";
```

Comment:
the SHA256 hash is the same for 1.6.2 and 1.4.2. Is this normal?

Reply:
It's my lazyness to compute hashes locally. The CI did it for me :see_no_evil: